lib/bolt/executor.rb

```diff
@@ -18,14 +18,18 @@ class Executor
     attr_reader :noop, :transports
     attr_accessor :run_as
 
+    # FIXME: There must be a better way
+    # https://makandracards.com/makandra/36011-ruby-do-not-mix-optional-and-keyword-arguments
     def initialize(concurrency = 1,
                    analytics = Bolt::Analytics::NoopClient.new,
                    noop = nil,
-                   bundled_content: nil)
+                   bundled_content: nil,
+                   load_config: true)
```

Comment:
Is there a test we can add around running a task with executor configured with `load_config: false`? Maybe expect that some Net::SSH setup isn't called?

Reply:
+1. I added a unit test to ssh_spec to verify this works! It doesn't ensure the var gets passed from executor, but it does ensure that the Net::SSH config isn't used.